File: draw me a chessboard.py
def chess_board(rows, columns):
    result = []
    for i in range(rows):
        if i%2==0:
            result.append(["O" if j%2==0 else "X" for j in range(columns)])
        else:
            result.append(["X" if j%2==0 else "O" for j in range(columns)])
    return result

File: draw me a chessboard/draw me a chessboard.js
function chessBoard(rows, columns) {
  let arr = [];
  for(let i=0;i<rows;i++){
    let tmp = []
    for(let j=0;j<columns;j++){
      if(i%2==0){
        if(j%2==0){
          tmp.push("O")
        }else{
          tmp.push("X")
        }
        
      }else{
        if(j%2==0){
          tmp.push("X")
        }else{
          tmp.push("O")
        }
      }
    }
    arr.push(tmp)
  }
  return arr
}
